src/nvim/usercmd.c

```diff
@@ -1532,10 +1532,12 @@ static size_t uc_check_code(char *code, size_t len, char *buf, ucmd_T *cmd, exar
   case ct_RANGE:
   case ct_COUNT: {
     char num_buf[20];
-    int64_t num = (type == ct_LINE1) ? eap->line1 :
-                  (type == ct_LINE2) ? eap->line2 :
-                  (type == ct_RANGE) ? eap->addr_count :
-                  (eap->addr_count > 0) ? eap->line2 : cmd->uc_def;
+    int64_t num = (type == ct_LINE1) ? eap->line1
```

Comment:
if we manaually break the line just before the `?` instead, would uncrustify keep that? would decrease the accumulating indentation.